test/functional/test_framework/test_framework.py

```diff
@@ -387,7 +387,7 @@ def _initialize_chain(self):
             # Create cache directories, run bitcoinds:
             for i in range(MAX_NODES):
                 datadir = initialize_datadir(self.options.cachedir, i)
-                args = [os.getenv("BITCOIND", "bitcoind"), "-server", "-keypool=1", "-datadir=" + datadir, "-discover=0"]
+                args = [os.getenv("BITCOIND", "bitcoind"), "-server", "-keypool=1", "-datadir=" + datadir, "-discover=0",  "-bind=127.0.0.1"]
```

Comment:
I'd prefer if you put it into the `bitcoin.conf` file (via `initialize_datadir`). This way, the option is valid also when nodes are (later) spun up for debugging purposes from outside the test framework.

Reply:
It's also shorter. Done.